packages/pwa/src/screens/FeedItemScreen.tsx

```
@@ -12,6 +12,7 @@
 import type {FeedItem, FeedItemId} from '@shared/types/feedItems.types';
 import {FeedItemContentType} from '@shared/types/feedItems.types';
 import {SystemTagId} from '@shared/types/tags.types';
+import {NavItemId} from '@shared/types/urls.types';
```

ESLint

[error] 'NavItemId' is defined but never used.

(unused-imports/no-unused-imports)